backend/sirius-web-diagrams/src/main/java/org/eclipse/sirius/web/diagrams/description/DiagramDescription.java

```diff
@@ -148,6 +158,11 @@ public Builder label(String label) {
             return this;
         }
 
+        public Builder autoLayout(boolean autoLayout) {
+            this.autoLayout = Objects.requireNonNull(autoLayout);
```

Comment:
`requireNonNull` is not needed on a bo`boolean`. It works thanks to auto-boxing, but it will create a `Boolean` instance, test that it is not `null` and then convert it back to a plain `boolean`.

Reply:
Done